### ISSUE TITLE
New-snmpv3user and new-snmpTrapDestination

### ISSUE THREAD

**Issue by DungKHoang** [HewlettPackard/POSH-HPOneView#447]
### Expected Behavior
Being able to create a snmpTrap with snmpv3 user

### Actual Behavior
Error due to type mismatch between "HPOneView.Networking.SnmpV3User" to and "HPOneView.Appliance.SnmpV3User".

### Steps to reproduce

```PowerShell
$userName                   = "dung-v3"
$securityLevel              = "None"
$snmpv3User1                = New-HPOVSnmpV3User -Username $userName -SecurityLevel $securityLevel
$inform                     = "Trap"
$trapDestinations           += New-HPOVSnmpTrapDestination -destination $destination -Port $port -SnmpV3User $snmpv3User1 -NotificationType $inform
[2001270745_HPOV_transcript.log](https://github.com/HewlettPackard/POSH-HPOneView/files/4117908/2001270745_HPOV_transcript.log)

```

### Version Information
HPE OneView PowerShell Library Version (`Get-HPOVVersion` or `$PSLibraryVersion`):
HPE OneView Appliance Version (`Get-HPOVVersion -ApplianceVer`):  
Output from `$PSVersionTable` on your Windows Host:

```PowerShell
# Output from $PSVersionTable
```

**Comment by ChrisLynchHPE** [comment]
This will be fixed in the next update.  The issue was with the parameter declaration inadvertently set to the wrong class type.

**Comment by ChrisLynchHPE** [comment]
This was already fixed in a newer 5.00 library update.

Closing due to no further activity.  If you wish, feel free to re-open to discuss further.
